README.md

```diff
@@ -55,6 +55,29 @@ It will produce this
 <p>Follow me on twitter!</p>
 ```
 
+## Options
+
+```js
+var md = require('markdown-it')()
+            .use(require('markdown-it-include'), [, options]);
+```
+
+* Type: `String|Object`
+
+If it's a string, it's the same as `options.root`.
+
+### root
+* Type: `String`
+* Default: `.`
+
```

Comment:
Here I would add a description of what is the meaning of `root`.

> `root` is the base directory of all the mardowns.

Reply:
Yeah~